### PR TITLE
feat: mgcovid19 components were updated in order to have tw clasess f…

### PR DESCRIPTION
**Pull request by roger-in-kiva**
MGCOVID19 components were updated in order to have Tailwind classes for typography tags and css options.

### CODE REVIEW

**Review by ryan-ludwig — APPROVED** [review]
Looks good!

File: src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
 	<div class="faq">
-		<h2 class="faq__header">
+		<h2 class="tw-mb-2">
 			Frequently asked questions
 		</h2>
 		<!-- eslint-disable max-len -->
 		<ul class="faq__list">
 			<li>
-				<h4>Are my contributions loans or donations?</h4>
+				<h3>Are my contributions loans or donations?</h3>
```
Comment (mcstover) [inline src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue:9]: @ryan-ludwig @roger-in-kiva  is this the same way we've handled FAQs on the other MG page?

**Review by mcstover — APPROVED** [review]
Nice cleanup and conversion!

File: src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue (hunk 1 above)
Comment (ryan-ludwig) [reply src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue:9]: No, most FAQs are powered by Contentful and use `<kv-frequently-asked-questions>`

File: src/pages/LandingPages/MGCovid19/MGCovidExplained.vue [hunk 2]
```diff
@@ -66,10 +66,10 @@
 							loading="lazy"
 						>
 					</div>
-					<h3 class="three-steps__title">
+					<h3 class="tw-mb-1">
 						Grow
 					</h3>
-					<div class="three-steps__description">
+					<div class="text-center">
```
Comment (eddieferrer) [inline src/pages/LandingPages/MGCovid19/MGCovidExplained.vue:72]: should this be tw-text-center ?

File: src/pages/LandingPages/MGCovid19/MGCovidExplained.vue [hunk 3]
```diff
@@ -45,16 +45,16 @@
 							loading="lazy"
 						>
 					</div>
-					<h3 class="three-steps__title">
+					<h3 class="tw-mb-1">
 						Check
 					</h3>
-					<div class="three-steps__description">
+					<div class="text-center">
```
Comment (emuvente) [inline src/pages/LandingPages/MGCovid19/MGCovidExplained.vue:51]: Should this be tw-text-center as well?

Comment (ryan-ludwig) [reply src/pages/LandingPages/MGCovid19/MGCovidExplained.vue:51]: Yes, I should do a find-replace on these.

Comment (ryan-ludwig) [reply src/pages/LandingPages/MGCovid19/MGCovidExplained.vue:51]: It works because of a foundation util class right now I think.

Comment (emuvente) [reply src/pages/LandingPages/MGCovid19/MGCovidExplained.vue:51]: 👍

**Review by emuvente — APPROVED** [review]
Awesome!

**Pull request merged**